Backpack/Font/Classes/Generated/BPKFont.h

```diff
@@ -21,74 +21,194 @@ typedef NS_ENUM(NSUInteger, BPKFontStyle) {
 
     BPKFontStyleTextBase = 0,
     BPKFontStyleTextBaseEmphasized = 1,
+    BPKFontStyleTextCaps = 11,
+    BPKFontStyleTextCapsEmphasized = 12,
     BPKFontStyleTextLg = 2,
     BPKFontStyleTextLgEmphasized = 3,
     BPKFontStyleTextSm = 4,
     BPKFontStyleTextSmEmphasized = 5,
     BPKFontStyleTextXl = 6,
     BPKFontStyleTextXlEmphasized = 7,
+    BPKFontStyleTextXlHeavy = 10,
     BPKFontStyleTextXs = 8,
     BPKFontStyleTextXsEmphasized = 9,
+    BPKFontStyleTextXxl = 13,
+    BPKFontStyleTextXxlEmphasized = 14,
+    BPKFontStyleTextXxlHeavy = 15,
+    BPKFontStyleTextXxxl = 16,
+    BPKFontStyleTextXxxlEmphasized = 17,
+    BPKFontStyleTextXxxlHeavy = 18,
 };
 
 NS_ASSUME_NONNULL_BEGIN
 @class UIFont;
 /**
  * BPKFont is the entry point for the Backpack typography stack. It expose the
  * supported text styles as static methods.
+ *
 */
 NS_SWIFT_NAME(Font) @interface BPKFont: NSObject
-+ (UIFont * _Nullable)fontWithStyle:(BPKFontStyle)style  NS_SWIFT_NAME(makeFont(style:));
+/*
+ * @warning This method is deprecated and using the font does not correctly render text with the Skyscanner style.
+ */
++ (UIFont * _Nullable)fontWithStyle:(BPKFontStyle)style  NS_SWIFT_NAME(makeFont(style:)) DEPRECATED_MSG_ATTRIBUTE("Use `attributedStringWithFontStyle:content` instead.");
 
 /**
  * The `textBase` text style from the Backpack typograhpy stack.
+ *
+ * @warning This method is deprecated and using the font does not correctly render text with the Skyscanner style.
 */
-+ (UIFont *)textBase;
++ (UIFont *)textBase DEPRECATED_MSG_ATTRIBUTE("Use `attributedStringWithFontStyle:content` instead.");
 
 /**
  * The `textBaseEmphasized` text style from the Backpack typograhpy stack.
+ *
+ * @warning This method is deprecated and using the font does not correctly render text with the Skyscanner style.
 */
-+ (UIFont *)textBaseEmphasized;
++ (UIFont *)textBaseEmphasized DEPRECATED_MSG_ATTRIBUTE("Use `attributedStringWithFontStyle:content` instead.");
+
+/**
+ * The `textCaps` text style from the Backpack typograhpy stack.
```

Comment:
I had to find _something_ to criticise 😉 

```suggestion
 * The `textCaps` text style from the Backpack typography stack.
```